torch/multiprocessing/spawn.py

```diff
@@ -118,7 +118,39 @@ def join(self, timeout=None):
         raise Exception(msg)
 
 
-def spawn(fn, args=(), nprocs=1, join=True, daemon=False):
+class SpawnContext(ProcessContext):
+    def __init__(self, processes, error_queues):
+        warnings.warn('SpawnContext is renamed to ProcessContext since 1.4 release.')
+        super(SpawnContext, self).__init__(self, processes, error_queues)
+    pass
+
+
+def start_processes(fn, args=(), nprocs=1, join=True, daemon=False, start_method='spawn'):
```

Comment:
Is this meant to be public? If so, maybe add doc entries and move the docstring to here?